.pep8speaks.yml

```diff
@@ -0,0 +1,7 @@
+# Repository configuration for pep8speaks GitHup App
+
+scanner:
+    linter: pycodestyle
+
+pycodestyle:  # Same as scanner.linter value. Other option is flake8
+    max-line-length: 100  # Default is 79 in PEP 8
```

Comment:
We have limit for 99 characters.
https://github.com/rhinstaller/anaconda/blob/master/CONTRIBUTING.rst#code-conventions